terraform/cluster/aws/main.tf

```diff
@@ -144,18 +144,30 @@ resource "aws_launch_template" "cluster" {
 }
 
 module "ebs_csi_driver_controller" {
+  depends_on = [
+    null_resource.wait_k8s_api
+  ]
+
   source  = "DrFaust92/ebs-csi-driver/kubernetes"
   version = "3.3.1"
 
   ebs_csi_controller_image                   = "public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver"
-  ebs_csi_driver_version                     = var.arm_type ? "v1.9.0-linux-arm64-amazon" : "v1.9.0"
+  ebs_csi_driver_version                     = "v1.9.0"
```

Comment:
`v1.9.0-linux-arm64-amazon` tag doesn't exist anymore, changing to `v1.9.0` (which is multi-arch, it will work on amd64 and arm64)